generators/app/templates/.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
+	"env": {
+		"browser": true,
+    "es6": true,
+    "jquery": true,
+    "node": true,
+    "jest": true
+	},
+	"extends": [
+    "eslint:recommended",
+    "prettier"
+  ],
+  "plugins": [
+    "import",
+    "prettier"
+  ],
+  "parserOptions": {
+    "sourceType": "module"
+  },
+	"rules": {
```

Comment:
These rules look great, but it would be nice to haave some docs, either in the readme or linked to it :)

Reply:
Okay, thanks for the suggestion. Can I commit more work in the same PR, please help me understand how do I push those changes after complete?

Reply:
Done.